### ISSUE TITLE
[SNES] Graphical Glitches in NHL 94

### ISSUE THREAD

**Issue by microace** [higan-emu/ares#347]
On 2022-02-09 nightly, there are graphical glitches in the NHL 94 opening screen:

![NHL '94 (USA) 2022-02-09 11-45-03](https://user-images.githubusercontent.com/3731236/153249417-be3a6800-d40f-4562-8a07-e9498d39d01b.png)

Expected behavior from bsnes:

![NHL '94 (USA)-001](https://user-images.githubusercontent.com/3731236/153249578-efca5177-0cb2-47e9-b21c-f1a077ad0e03.png)

Using redump NHL 94 USA rom

OS is Windows 10 with OpenGL 3.2 video driver

**Comment by Max833** [comment]
The reason why it is working with bsnes is the use of the scanline-based PPU renderer (you have to enable "fast mode"; ares has this renderer as well by default, but it's a little bit different) **and** this hack:
https://github.com/bsnes-emu/bsnes/blob/f57657f27ddec337b1960c7ddaa1b23894bc00c3/bsnes/target-bsnes/program/hacks.cpp#L44

If you use the cycle-based PPU renderer with bsnes (you have to disable "fast mode") the same problem appears. **_But_** for a different reason: https://arstechnica.com/gaming/2021/06/how-snes-emulators-got-a-few-pixels-from-complete-perfection/

**Long story short:** There are still many games like that (with very minor issues). Decapping the PPUs is the only 100% solution to solve that problem.
